code/datums/shuttles.dm

```diff
@@ -11,6 +11,7 @@
 
 	var/credit_cost = INFINITY
 	var/can_be_bought = TRUE
+	var/syndicate_exclusive = FALSE	//makes you able to buy the shuttle in emaged comms console even if can_be_bought is FALSE
```

Comment:
grammar correction, should be.
//makes you able to buy the shuttle at a emagged comms console even if can_be_bought is FALSE